tests/test_ommx_gurobipy.py

```diff
@@ -56,3 +56,49 @@ def test_multi_objective_handling():
     # Should maximize x to its upper bound
     assert solution.state.entries[1] == pytest.approx(1)
     assert solution.objective == pytest.approx(1)
+
+
+def test_partial_evaluate():
+    """Test that the adapter correctly handles partially evaluated instances with used_decision_variables"""
+    x = [DecisionVariable.binary(i, name="x", subscripts=[i]) for i in range(3)]
+    instance = Instance.from_components(
+        decision_variables=x,
+        objective=x[0] + x[1] + x[2],
+        constraints=[(x[0] + x[1] + x[2] <= 1).set_id(0)],  # one-hot constraint
```

Comment:
```suggestion
        constraints=[(x[0] + x[1] + x[2] <= 1).set_id(0)],
```

We should remove the comment because this constraint is not a one-hot constraint.